public/javascripts/encryption.js

```diff
@@ -0,0 +1,15 @@
+const bcrypt = require('bcryptjs');
+
+// Function hashes a user imputed password
+exports.encryptPassword = (password) => {
+
+    return(bcrypt.hashSync(password, 10));   
+};
+
+// Function compares user imputed password to hashed password in database
+exports.decryptPassword = (password, passwordHash) => {
```

Comment:
Since `bcrypt` is a hashing library, there's actually no decryption taking place (hashing is a 1-way process: Once you hash a password, you cannot un-hash it). This function then is not decryption, but comparison.

The reason is that hashes, while they cannot be reversed, are determinate. Hashing the same word will return the same hash every time. So, by hashing the inputted password and comparing to the value stored in the DB, we can determine if the entered password is the same as the one stored in the database.

All that to say is that this method would be better named as `comparePassword` or similar